lib/plurimath/math/function/underover.rb

```diff
@@ -50,6 +50,27 @@ def to_omml_without_math_tag(display_style)
 
           underover(display_style)
         end
+
+        def line_breaking(obj)
+          parameter_one&.line_breaking(obj)
+          if obj&.exist?
```

Comment:
rename `exist?` to something more meaningful because we are checking for existence of the value not the whole object here.